src/types-and-traits.rst

```diff
@@ -2693,6 +2693,30 @@ A :t:`higher-ranked trait bound` is a :t:`bound` that specifies an infinite
 list of :t:`[bound]s` for all possible :t:`[lifetime]s` specified by the
 :s:`ForGenericParameterList`.
 
+:dp:`fls_J9DEsd06Ttu9`
+An :t:`implied bound` is a :t:`bound` that is not expressed in syntax, but is
+is the byproduct of relations between :t:`[lifetime parameter]s` and
+:t:`[function parameter]s`, between :t:`[lifetime parameter]s` and a
+:t:`return type`, and between :t:`[lifetime parameter]s` and :t:`[field]s`.
+
+is the byproduct of relations between the :t:`[generic parameter]s` of
+:t:`[abstract data type]s` or :t:`[function]s`.
+
+:dp:`fls_IfHRxSasGAih`
+A :t:`reference` of the form ``&'lifetime Type``, where ``'lifetime`` is a
+:t:`lifetime parameter` and ``Type`` is a :t:`type`, yields :t:`implied bound``
+``Type: 'lifetime``.
+
+:dp:`fls_K8nPGP5xbLb7`
+If an explicit :t:`bound` or an :t:`implied bound` applies to the :t:`type` of
+a :t:`field`, then this abound also applies to the related
+:t:`abstract data type` as an :t:`implied bound`.
```

Comment:
only outlives bounds matter, that is bounds where the right side has lifetimes (opposed to traits), other bounds are never implied (so `T: 'a` and `'a: 'b` bounds)

Reply:
Fixed.